src/components/App.jsx

```diff
@@ -71,7 +76,7 @@ class App extends React.Component {
       Hi friends!
       npm run react-dev should open a live listener of webpack,
       in another terminal do npm run server-dev and navigate to localhost:8000 to view the app!
-      <div><ProductDetails
+      <div><ProductDetails updateStyle={this.updateStyle}

```

Comment:
I have defined the function as this.updateStyle = this.updateStyle.bind(this) elsewhere in this doc so it will function as having .bind(this).